contracts/ToucanRegenBridge.sol

```diff
@@ -90,14 +85,19 @@ contract ToucanRegenBridge is Ownable, Pausable, AccessControl {
     //      Constructor
     // ----------------------------------------
 
-    constructor(address tokenIssuer_, INCTPool nctPool_) Ownable() {
-        tokenIssuer = tokenIssuer_;
+    constructor(
+        INCTPool nctPool_,
+        bytes32[] memory roles,
+        address[] memory accounts
+    ) {
+        require(accounts.length == roles.length, "accounts and roles must have same length");
         nctPool = nctPool_;
-        _grantRole(PAUSER_ROLE, msg.sender);
-        if (tokenIssuer_ != address(0)) {
-            _grantRole(PAUSER_ROLE, tokenIssuer_);
-            emit TokenIssuerUpdated(address(0), tokenIssuer_);
+        uint256 adminAccounts;
+        for (uint256 i = 0; i < accounts.length; ++i) {
+            _grantRole(roles[i], accounts[i]);
+            if (roles[i] == DEFAULT_ADMIN_ROLE) adminAccounts += 1;
         }
+        require(adminAccounts == 1, "should have one admin role");
```

Comment:
Probably better not to restrict this to one admin
```suggestion
        require(adminAccounts >= 1, "should have at least one admin");
```
Could even simplify `adminAccounts` to a boolean

Reply:
> Probably better not to restrict this to one admin

That's okay, I got the idea from your previous comment that we only want a single admin, but I can easily modify this. Thanks for the clarification.